DayrSuryan/DSEthiop23.xml

```diff
@@ -27,50 +27,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                        <textLang mainLang="en" otherLangs="ti"/>
+                        <note>Foreword in English (printed)  and Tigrinya on the composition of the Tigrinya translation of the
+                           Śǝrʿāta qǝddāse.</note>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i4.2">
+                        <locus from="1r"/>
+                        <title type="complete" ref="LIT5910QeddaseTi#SerataQ"/>
+                        <textLang mainLang="ti"/>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i4.3">
+                        <locus to="122r"/>
+                        <title type="complete" ref="LIT5910QeddaseTi#Qeddase"/>
+                        <textLang mainLang="ti"/>
+                        
+                        <msItem xml:id="ms_i4.3.1">
+                           <title type="complete">Foreword</title>
+                           <textLang mainLang="en" otherLangs="ti"/>
+                           <note>Foreword in English and Tigrinya on the two folios preceding <locus target="#66"/>.</note>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i4.3.2">
+                           <locus from="66r" to="122r"/>
+                           <title type="complete">Nāy ʾamānǝyān qǝddāse</title>
+                           <textLang mainLang="gez" otherLangs="ti"/>
+                           <note>The text is in Gǝʿǝz with Tigrinya translation.</note>
+                           
+                           <incipit  xml:lang="ti"> ናይ፡  አማንያን፡  ቅዳሴ፡  አብዚ፡  ይጅመር፡  ዲያቆን፡  ፃኡ፡  ንኡሰ፡  ክርስቲያን፡  ዘይጠመቕኩም፡  ዉጹ።
+                              ካህን፡  አብዚ፡  ብድጋም፡  ዝብሎ፡  አሎ፡  ዲያቆን፡  ተንሥኡ፡  ለጸሎት፡ ፡  ንጸሎት፡  ተንሥኡ፡</incipit>
+                              
+                              <explicit xml:lang="ti">
+                                 <locus target="#121r"/> ናይ፡  ጐይታና፡  የሱስ፡  ክርስቶስን፡  ፍቕሪ፡  እግዚአብሔርን፡  ኅብረት፡  መንፈስ፡  ቅዱስን፡  ምስ፡  ኵላትኩም፡  
+                                 ይኩን፡  (፪ቆርንቶስ፡  ፲፫፡  ፲፬)  ሕዝቢ፡  አሜን፡   
+                              </explicit>
+                        </msItem>
+                     </msItem>
                   </msItem>
 
+            
                   <msItem xml:id="ms_i5">
-                     <title type="complete">ʾĀndi ḥǝggi nāy Gǝbṣi ʾortodoks beta krǝstiyān</title>
+                     <locus from="1" to="24r"/>
+                     <title type="complete" ref="LIT5911Heggi"/>
                      <textLang mainLang="ti"/>
+                     <note>The text is the collection of the statutes of the Coptic Church
+                        approved on <date>2 June 1985</date> (the feast of <persName ref="PRS4306GabraMa"/>). 
+                        The statutes were issued in Arabic and English; the translator, ʾabuna Maqaryos, used for his translation the Arabic version.
```

Comment:
Maqāryos

Reply:
It seems that it was there, and I transcribed. So, still Maqaryos emerges on the surface.